app/Models/Book.php

```diff
@@ -4,13 +4,16 @@
 
 use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
+
 //use Legalworks\IsbnTools\IsbnCast;
 
 class Book extends Model
 {
     use HasFactory;
-    protected $table="book";
-    protected $primarykey="id";
-    protected $fillable=["book_name","author_name","book_ibsn","image"];
-  //  protected $casts = ["book_ibsn" => IsbnCast::class];
+    protected $fillable=["id","name","author_id","isbn","image"];
```

Comment:
13. satırda her sütun için teker teker fillable a yazacağına. protected $guarded=[]; yazarsan bütün sutünlar yazılabilir olur